### ISSUE TITLE
No concurrency support

### ISSUE THREAD

**Issue by zmeyc** [sebastian-software/effective-prettier#4]
Docs mentions `multi-threading with glob-based file lists`, but unless I'm missing something it doesn't seems to be implemented. Afaik PQueue is single-threaded (sequential) and promise-based. For concurrency support, [node worker threads](https://nodejs.org/api/worker_threads.html) need to be used.

**Comment by swernerx** [comment]
You are right. The docs might need to get a little tweak here. It executes in parallel... so it at least does not have negative side-effects with blocking IO. I have no resources right now to work on node workers... that might indeed be possible though.
